pynamodb/settings.py

```diff
@@ -0,0 +1,36 @@
+from os import getenv
```

Comment:
https://www.python.org/dev/peps/pep-0008/#imports

```
Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.
```

there is a tool called `isort` that can do it for you automatically too